specs/adr/schema_ADR.md

```diff
@@ -0,0 +1,50 @@
+# Schema Format
+
+* Status: proposed 
+* Deciders: Alvin Wang
+* Date: 2021-05-20 
+
+
+## Context and Problem Statement
+
+The following is documenting the decisions we made about how we formatted the schema and the reasons behind it
+
+## Decision Drivers
+
+* Implementing a schema model to account for all the information we need for the bullet journal
+
+## Considered Options
+
+* Current schema
+
+## Decision Outcome
+
+* Current schema
+
+### Positive Consequences
+
+* Schema includes all information required for bullet journal, including name, collections, and daily logs.
+
+### Negative Consequences
+
+* May be a bit large as there are lots of moving parts.
```

Comment:
We should also add that with the chosen schema, it is difficult to retain the order in which tasks/notes/events were added to the comprehensive view. We accepted this limitation when creating the relevant issue during last sprint

Reply:
@alvinwang922 when possible make this revision; the second checkpoint for ADR's is due by tomorrow.

Reply:
Nevermind, I did it.